ethers-middleware/src/gas_oracle/etherchain.rs

```diff
@@ -1,32 +1,63 @@
-use ethers_core::types::U256;
-
+use super::{from_gwei_f64, GasCategory, GasOracle, GasOracleError, Result};
 use async_trait::async_trait;
+use ethers_core::types::U256;
 use reqwest::Client;
 use serde::Deserialize;
 use url::Url;
 
-use crate::gas_oracle::{GasCategory, GasOracle, GasOracleError, GWEI_TO_WEI};
-
-const ETHERCHAIN_URL: &str = "https://www.etherchain.org/api/gasPriceOracle";
+const URL: &str = "https://www.etherchain.org/api/gasPriceOracle";
```

Comment:
https://twitter.com/milancermak/status/1604441584808103936 maybe we can use this https://etherchain.org/api/gasnow, which redirects to https://beaconcha.in/api/v1/execution/gasnow